prql-java/prql-jdbc/README.md

```diff
@@ -0,0 +1,23 @@
+# PRQL JDBC Driver
+
+JDBC wrapper taking PRQL as input.
+
+## Maven
+
+```xml
+<dependency>
+    <groupId>org.prqllang</groupId>
+    <artifactId>prql-jdbc</artifactId>
+    <version>${PRQL_VERSION}</version>
+</dependency>
+```
+
+## Get Started
+
+`jdbc:prql:<rest of the connection string>`
+
+## Supported Database
+
+| Database   | Dialect    | JDBC Driver | Connection String             |
+| ---------- | ---------- | ----------- | ----------------------------- |
+| ClickHouse | clickhouse |             | jdbc:prql:ch:http://localhost |
```

Comment:
Is this the only DB that's currently supported, or it's a draft list?

Reply:
Sorry I was too lazy as I only tried on ClickHouse and SQLite. It should work for all JDBC-compatible databases. It's going to be a long list but I'll document a few more after testing.

Reply:
No prob! Fine to leave it as a TODO! As long as we're transparent about the current state it's fine to make incremental progress...